### PR TITLE
select rows

### PR DESCRIPTION
**Pull request by Ahmedoudouh**
i made select perpage but i had error if i selected 15 give me 10 rows

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -272,14 +272,15 @@ function App() {
     protect: "unlock",
```
Comment (faouzioudouh) [inline src/App.jsx:272]: Move array of customers to different file.

File: src/App.jsx [hunk 2]
```diff
@@ -272,14 +272,15 @@ function App() {
...
     select: "unchecked",
   }];
-
+  
   const [customersList, setcustomersList] = useState(customers)
```
Comment (faouzioudouh) [inline src/App.jsx:276]:
```suggestion
  const [customersList, setCustomersList] = useState(customers)
```

File: src/App.jsx [hunk 3]
```diff
@@ -272,14 +272,15 @@ function App() {
...
-
   return (
+    
     <div>
-      <Header array={customersList} arrayOriginal={customers} search={setcustomersList} />
+      
+      <Header  arrayOriginal={customers} search={setcustomersList} />
```
Comment (faouzioudouh) [inline src/App.jsx:281]:
```suggestion
      <Header  customers={customers} setCustomersList={setCustomersList} />
```

File: src/App.jsx [hunk 4]
```diff
@@ -272,14 +272,15 @@ function App() {
...
       <Table array={customersList} setcustomersList={setcustomersList} />
```
Comment (faouzioudouh) [inline src/App.jsx:282]:
```suggestion
      <Table customers={customers} setCustomersList={setCustomersList} />
```

File: src/App.jsx [hunk 5]
```diff
@@ -272,14 +272,15 @@ function App() {
...
-      <Footer array={customersList} />
+      <Footer array={customersList } render={setcustomersList}/>
```
Comment (faouzioudouh) [inline src/App.jsx:283]:
```suggestion
      <Footer customers={customers} setCustomersList={setCustomersList} />
```

File: src/Footer/Footer.jsx [hunk 6]
```diff
@@ -5,12 +5,15 @@ import PerPage from "./PerPage/Per-page.jsx"
 import NextPage from "./NextPage/Next-page.jsx"
 import PreviousPage from "./PreviousPage/Previous-page.jsx"
 
-export const Footer = ({ array }) => {
+export const Footer = ({ array,render }) => {
+
+ 
+
```
Comment (faouzioudouh) [inline src/Footer/Footer.jsx:11]: 3 empty lines?

File: src/Footer/Footer.jsx [hunk 7]
```diff
@@ -5,12 +5,15 @@ import PerPage from "./PerPage/Per-page.jsx"
 import NextPage from "./NextPage/Next-page.jsx"
 import PreviousPage from "./PreviousPage/Previous-page.jsx"
 
-export const Footer = ({ array }) => {
+export const Footer = ({ array,render }) => {
```
Comment (faouzioudouh) [inline src/Footer/Footer.jsx:8]:
```suggestion
export const Footer = ({ customers, setCustomers }) => {
```

File: src/Footer/Footer.jsx [hunk 8]
```diff
@@ -5,12 +5,15 @@ import PerPage from "./PerPage/Per-page.jsx"
...
+
+ 
+
     return (
         <footer className="content footer">
             <Active ActiveCustomers={array} />
             <div className="group-footer">
-                <PerPage />
+                <PerPage array={array}  render={render} />
```
Comment (faouzioudouh) [inline src/Footer/Footer.jsx:16]:
```suggestion
                <PerPage customers={customers}  setCustomers={setCustomers} />
```

File: src/Footer/PerPage/Per-page.jsx [hunk 9]
```diff
@@ -1,19 +1,28 @@
...
-function PerPage() {
+    const [selectValue, setselectValue] = useState()
+    let copieArr=[...array].slice(0, selectValue)
+    const selectValueRows = e => {
+        setselectValue(e.target.value)
+        render(copieArr)
+    };
+    
+    console.log(selectValue)
+    //console.log(array)
```
Comment (faouzioudouh) [inline src/Footer/PerPage/Per-page.jsx:13]: WHAT IS THIS?

File: src/Footer/PerPage/Per-page.jsx [hunk 10]
```diff
@@ -1,19 +1,28 @@
...
-                    <option value="5" >5</option>
+                <select onChange={selectValueRows} id="select" className="style-select" >
                     <option value="10">10</option>
                     <option value="15">15</option>
                     <option value="20">20</option>
                     <option value="25">25</option>
                 </select>
             </span>
-        </div>
+        </div >
```
Comment (faouzioudouh) [inline src/Footer/PerPage/Per-page.jsx:25]:
```suggestion
        </div>
```

File: src/Input/Input-search.jsx [hunk 11]
```diff
@@ -19,7 +19,6 @@ export const Input = ({ filteredFunction, arrayOriginal }) => {
     setsearch(e.target.value);
     filteredFunction(filteredPersons)
   };
-
```
Comment (faouzioudouh) [inline src/Input/Input-search.jsx:22]: Keep empty line between functions

File: src/Tbody/Icon/Icon.jsx [hunk 12]
```diff
@@ -3,7 +3,7 @@ import "./Style-icon.css";
 export const Icon = ({ array, deleteWhenClick }) => {
     return (
         <div>
-            <img onClick={() => deleteWhenClick(array.id)} className="icon padding" id="combinedSort" src="Images/trash.png" />
+            <img onClick={() => deleteWhenClick(array.number)} className="icon padding" id="combinedSort" src="Images/trash.png" />
```
Comment (faouzioudouh) [inline src/Tbody/Icon/Icon.jsx:6]: What is `array.number` ?

File: src/Tbody/Tbody.jsx [hunk 13]
```diff
@@ -11,22 +11,24 @@ import Inr from "./Inr/Inr.jsx";
...
-            customer.id === customerId
+            customer.number === customerId
         })
         newArray.splice(index, 1)
+      
         setcustomersList(newArray)
+      
     }
-
+  */
```
Comment (faouzioudouh) [inline src/Tbody/Tbody.jsx:25]: What is this?

**Review by faouzioudouh — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Ahmedoudouh) [reply src/App.jsx:272]: i dont understand well why i need to move to another file

Comment (faouzioudouh) [reply src/App.jsx:272]: To keep your component simple.

File: src/Footer/PerPage/Per-page.jsx (hunk 9 above)
Comment (Ahmedoudouh) [reply src/Footer/PerPage/Per-page.jsx:13]: function to select rows perpage but have error

**Pull request merged**